go/vt/vttablet/tabletmanager/vreplication/table_plan_builder.go

```diff
@@ -669,9 +675,17 @@ func (tpb *tablePlanBuilder) generateValuesPart(buf *sqlparser.TrackedBuffer, bv
 		separator = ","
 		switch cexpr.operation {
 		case opExpr:
-			if cexpr.colType == querypb.Type_JSON {
+			switch cexpr.colType {
+			case querypb.Type_JSON:
 				buf.Myprintf("convert(%v using utf8mb4)", cexpr.expr)
-			} else {
+			case querypb.Type_DATETIME:
+				sourceTZ := tpb.source.GetSourceTimeZone()
+				if sourceTZ != "" {
+					buf.Myprintf("convert_tz(%v, '%s', '%s')", cexpr.expr, sourceTZ, "UTC")
+				} else {
+					buf.Myprintf("%v", cexpr.expr)
+				}
+			default:
```

Comment:
:+1:  Just a question: is `UTC` preferable to `+00:00`? No opinion myself.

Reply:
> +1 Just a question: is `UTC` preferable to `+00:00`? No opinion myself.

Based on my comment below about reverse streams, the UTC timezone will make it to the proto in the vreplication table. So it is probably better to leave the `UTC` timezone code in there rather than the `+00:00` offset as a nudge to avoid setting up the source time zones in terms of offsets (to avoid DST-related issues)